quirks/password-rules.json

```diff
@@ -326,6 +332,9 @@
     "marriott.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$!#&@?%=];"
     },
+    "maybank2u.com.my": {
+        "password-rules": "minlength: 8; maxlength: 12; max-consecutive: 2; required: lower; required: upper; required: digit; required: special;"
```

Comment:
Whoops, I missed one tidbit when I initially reviewed this!

The screenshot of the password rules for this site states:

```
Must NOT contain space
```

The `special` character class allows for spaces, though.

That means we'll have to explicitly specify the allowed special characters here (so we can omit space):

```
required: [-~!@#$%^&*_+=`|(){}[:;\"'<>,.?];
```

(I pulled that list of characters from [this page](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules).)

Reply:
Oops, looks like I misunderstood the following line from the [Apple Developer document](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules) then:
> You can combine these keywords to form your rule. **Duplicate property values**, empty character classes, and properties without a value are ignored.

Thanks for catching that, I'll fix it.

Reply:
Ahh, I see. I didn't realise that spaces were included under the character list in the "required: special" tag.